policy/modules/services/postfix.te

```diff
@@ -538,10 +541,14 @@ files_dontaudit_search_var(postfix_map_t)
 
 auth_use_nsswitch(postfix_map_t)
 
+domain_use_interactive_fds(postfix_map_t)
+
 logging_send_syslog_msg(postfix_map_t)
 
 miscfiles_read_localization(postfix_map_t)
 
+userdom_use_user_ptys(postfix_map_t)
+
```

Comment:
It's been quite some time since I used postfix... how is this used interactively?

Reply:
The sysadmin runs postmap to create a hashed or indexed file out of a plain text file.

Reply:
Why is this needed over having sysadm simply `execute_no_trans` on map and simply write the files?

Reply:
That works for me.

Reply:
For now I'll just remove this and I'll add something else for /etc/aliases in a different PR